src/api/app/views/staging/staging_projects/_missing_reviews.xml.builder

```diff
@@ -1,5 +1,5 @@
 builder.missing_reviews(count: count) do |missing_review|
   missing_reviews.each do |bs_request|
-    missing_review.entry(id: bs_request[:id], creator: bs_request[:by], state: bs_request[:state], package: bs_request[:package])
+    missing_review.entry(id: bs_request[:id], request: bs_request[:request], creator: bs_request[:by], state: bs_request[:state], package: bs_request[:package])
```

Comment:
@vpereira this is a left over of me debugging something and I'll remove it - but this should be fixed independently of it.

Because the id of the review is super uninteresting - there is nothing you can do with it, there is no API for reviews (unfortunatly one might add), but what's missing is the request number.